src/execution/fixtures/ExecuteAfm.fixtures.ts

```diff
@@ -425,6 +425,59 @@ const oneMeasureResponse: Execution.IExecutionResponses = {
     }
 };
 
+const oneMeasureOneDimensionResponse: Execution.IExecutionResponses = {
+    executionResponse: {
+        executionResponse: {
+            dimensions: [
+                {
+                    headers: [
+                        {
+                            measureGroupHeader: {
+                                items: [
+                                    {
+                                        measureHeaderItem: {
+                                            name: 'Lost',
+                                            format: '$#,##0.00',
+                                            localIdentifier: '1st_measure_local_identifier',
+                                            uri: '/gdc/md/d20eyb3wfs0xe5l0lfscdnrnyhq1t42q/obj/1283',
+                                            identifier: 'af2Ewj9Re2vK'
+                                        }
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                }
+            ],
+            links: {
+                // tslint:disable-next-line:max-line-length
+                executionResult: '/gdc/app/projects/d20eyb3wfs0xe5l0lfscdnrnyhq1t42q/executionResults/2651138797087227392'
```

Comment:
Its not necessary to use real link. Then you can get rid of tslint comment as well